ams/server/src/main/java/com/netease/arctic/server/table/executor/TableRuntimeRefreshExecutor.java

```diff
@@ -51,6 +53,17 @@ public void handleStatusChanged(TableRuntime tableRuntime, OptimizingStatus orig
     tryEvaluatingPendingInput(tableRuntime, loadTable(tableRuntime));
   }
 
+  @Override
+  public void handleConfigChanged(TableRuntime tableRuntime, TableConfiguration originalConfig) {
+    OptimizingConfig originalOptimizingConfig = originalConfig.getOptimizingConfig();
+    OptimizingConfig newOptimizingConfig = tableRuntime.getOptimizingConfig();
+    if (!originalOptimizingConfig.equals(newOptimizingConfig) &&
+        newOptimizingConfig.isEnabled()) {
```

Comment:
Would it be better to use
```
if (!originalOptimizingConfig.isEnable &&
        newOptimizingConfig.isEnabled())
```?

Reply:
If it is simply to see whether it is turned on or off, it is enough to write this way, but I don’t know if other configuration changes under other originalOptimizingConfig will affect subsequent operations

Reply:
I believe it's better to compare all optimizing configurations, because most configurations will affect the behavior of the plan.